modules/atomic-widgets/prop-types/union-prop-type.php

```diff
@@ -45,6 +45,26 @@ public function get_prop_type( $type ): ?Transformable_Prop_Type {
 		return $this->prop_types[ $type ] ?? null;
 	}
 
+	private function get_prop_type_from_value( $value ): ?Prop_Type {
+		if ( isset( $value['$$type'] ) ) {
+			return $this->get_prop_type( $value['$$type'] );
+		}
+
+		if ( is_numeric( $value ) ) {
+			return $this->get_prop_type( 'number' );
+		}
+
+		if ( is_bool( $value ) ) {
+			return $this->get_prop_type( 'boolean' );
+		}
+
+		if ( is_string( $value ) ) {
+			return $this->get_prop_type( 'string' );
+		}
```

Comment:
Should we remove it?

Reply:
Done + fixed a test which still had a shorthand value